components/cards/Cards.module.css

```diff
@@ -0,0 +1,52 @@
+.recipecard {
```

Comment:
As you are using the `Cards.module.css` it is already clear that you are describing classes for the cards component.
```suggestion
.recipe {
```

Reply:
would be fixed in next pullrequest. thank you